test/main.cpp

```diff
@@ -67,6 +67,12 @@ int main(int argc, char* argv[]) {
         exclude.insert("Perf");
     }
 
+    if (args.isSet("-10x")) {
```

Comment:
NAB: kind of weird name since it takes a number to repeat, it doesn't repeat 10 times the command. How about `-duplicateRequest`?